tests/test_cases/input/multiline_string_escape_newline_at_end.kdl

```diff
@@ -0,0 +1,4 @@
+node """
+    a
+   \
```

Comment:
can you add the `_fail` equivalent to this (where `a` would end up indented less than the resolved prefix)